compose/project.py

```diff
@@ -115,6 +117,14 @@ def from_dicts(cls, name, service_dicts, client, use_networking=False, network_d
                     net=net,
                     volumes_from=volumes_from,
                     **service_dict))
+        if config_data.volumes:
+            for vol_name, data in config_data.volumes.items():
+                project.volumes.append(
+                    Volume(
+                        client=client, project=name, name=vol_name,
+                        driver=data.get('driver'), driver_opts=data.get('driver_opts')
+                    )
```

Comment:
I think we will probably need to prefix the volume_name here with the project name, so that we continue to support multiple isolated environments on the same host (documented as one of the key features of compose http://docs.docker.com/compose/#Multiple-isolated-environments-on-a-single-host).